prov/shm/src/smr_util.h

```diff
@@ -225,8 +225,10 @@ struct smr_region {
 	int		pid;
 	uint8_t		cma_cap_peer;
 	uint8_t		cma_cap_self;
-	uint32_t	max_sar_buf_per_peer;
 	uint8_t		xpmem_cap_self;
+	uint8_t		resv2;
+
+	uint32_t	max_sar_buf_per_peer;
```

Comment:
Why not use `_Alignas()`?

```suggestion
	_Alignas(8) uint32_t	max_sar_buf_per_peer;
```

Reply:
Mostly because I didn't know about it :)
Can you elaborate on what that would do?
I personally like the explicit value because it's easier to see that extra byte but I'm open to suggestions if anyone feels strongly about it.

Reply:
@darrylabbate are you still pushing it?

Reply:
I guess it depends on the intent. If you want to reserve the byte `resv2` for future use, the explicit field makes more sense. Otherwise, [`_Alignas()`](https://en.cppreference.com/w/c/language/_Alignas) will set the alignment requirement of `max_sar_buf_per_peer` however you wish, without the hardcoded `uint8_t` fields littering the struct definition.

This is mainly useful since you don't need to worry about how various systems/architectures handle data type alignment (C standard makes few guarantees), and future devs don't need to guess what the random `uint8_t`s are for.

Reply:
> @darrylabbate are you still pushing it?

No this is not a blocker